rules/share-the-action-items-that-came-up/rule.md

```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
 title: After - Do you share the action items that came up after a meeting?
-uri: share-the-action-items-that-came-up
```

Comment:
1. Please do the other steps on https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Rename-Rules

Reply:
Hi @tiagov8 

All done

>1 - Please improve the PR title... "Updated rule.md" is not descriptive enough
>2 - You need to follow some extra steps to shorten a URL: https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Rename-Rules. Please action
>3 - Add me as reviewer ;)